docs/js/playground.js

```diff
@@ -1,7 +1,7 @@
-var HtmlGenerator = require('html-generator').HtmlGenerator
+var HtmlGenerator = require('../../src/html-generator').HtmlGenerator
 var en = require('hyphenation.en-us')
 
-var latexjs = require('latex-parser')
+var latexjs = require('../../src/latex-parser')
 
```

Comment:
I am not sure this is correct. Once the playground is deployed on github, there won't be access to `src` anymore - at the very latest when I switch to vuepress.